docs/plugins/google-cloud.md

```diff
@@ -134,22 +189,237 @@ This field specifies the timeout for the metrics export in milliseconds.
+| genkit/ai/generate/output/images     | Counter   | Output images                              |
+| genkit/ai/generate/input/audio       | Counter   | Input audio files                          |
+| genkit/ai/generate/output/audio      | Counter   | Output audio files                         |
+
+Each generate metric contains the following dimensions:
+
+| Name            | Description                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------- |
+| modelName       | The name of the model                                                                                |
+| featureName     | The name of the parent feature being executed                                                        |
+| path            | The path of execution from the feature root to this action. eg. '/myFeature/parentAction/thisAction' |
+| temperature     | The temperature parameter passed to the model                                                        |
+| maxOutputTokens | The maxOutputTokens parameter passed to the model                                                    |
+| topK            | The topK parameter passed to the model                                                               |
+| topP            | The topP parameter passed to the model                                                               |
+| latencyMs       | The response time taken by the model                                                                 |
+| status          | 'success' or 'failure' depending on whether or not the feature request succeeded                     |
+| error           | Only set when `status=failure`. Contains the error type that caused the failure                      |
+| source          | The Genkit source language. Eg. 'ts'                                                                 |
+| sourceVersion   | The Genkit framework version                                                                         |
+
+Visualizing metrics can be done through the Metrics Explorer. Using the left 
+hand side menu, click 'Metrics explorer' under the 'Explore' heading.
+
+<img src="../resources/cloud-ops-metrics-explorer.png" width="750">
+
+Select a metrics by clicking on the 'Select a metric' dropdown, selecting
+'Generic Node', 'Genkit', and a metric.
+
+<img src="../resources/cloud-ops-metrics-generic-node.png" width="750">
+
+The visualization of the metric will depend on its type (counter, histogram,
+etc). The Metrics Explorer provides robust aggregation and querying facilities
+to help graph metrics by their various dimensions.
+
+<img src="../resources/cloud-ops-metrics-metric.png" width="750">
+
+## Telemetry Delay
+
+There may be a slight delay before telemetry for a particular execution of a
+flow is displayed in Cloud's operations suite. In most cases, this delay is
+under 1 minute.
```

Comment:
Do we want to mention the export interval config setting here or no?

Reply:
not sure - I figured this was a backend/processing delay. but if not, we can update here. going to table for now.

Reply:
Oh yeah you right